Heine.Mvc.ActionFilters/Extensions/HeaderUtilities.cs

```diff
@@ -47,5 +48,32 @@ internal static string DumpHeaders(params HttpHeaders[] headers)
             stringBuilder.Append('}');
             return stringBuilder.ToString();
         }
+
+        internal static Dictionary<string, string> CloneHeaders(params HttpHeaders[] headersCollections)
```

Comment:
Not sure if I like the name `CloneHeaders`. Clone indicates that the method creates an identical copy, which in this case may not be true.

Reply:
Yeah, was thinking about that myself, but couldn't find a better name myself. Any suggestions?

Reply:
`GetLoggableHeaders`, `ObfuscateHeaders`, `RemovePrivateHeaders`, `RemoveSecretHeaders` ?

Reply:
Fixed